src/__tests__/08-specter.ts

```diff
@@ -66,46 +66,47 @@ it('createWallet', async () => {
 it('deposit to bitcoind', async () => {
   const initBitcoindBalance = await specterWallet.getBitcoindBalance()
   const { chain_balance: initLndBalance } = await getChainBalance({ lnd })
-  
+
   const sats = 10000
-  
+
   await specterWallet.toColdStorage({ sats })
   await bitcoindDefaultClient.generateToAddress(6, RANDOM_ADDRESS)
 
-  // TODO: use events, to make sure lnd has updated its utxo set 
+  // TODO: use events, to make sure lnd has updated its utxo set
   // and considered the change UTXO in the balance
   await sleep(1000)
-  
+
   const bitcoindBalance = await specterWallet.getBitcoindBalance()
   const { chain_balance: lndBalance } = await getChainBalance({ lnd })
 
   expect(bitcoindBalance).toBe(initBitcoindBalance + sats)
 
   const { results: [{ fee }] } = await MainBook.ledger({ account: bitcoindAccountingPath, type: "to_cold_storage" })
-  
+
   console.log({lndBalance, initLndBalance, sats, fee, bitcoindBalance, initBitcoindBalance})
   expect(lndBalance).toBe(initLndBalance - sats - fee)
 
 })
 
-it('withdrawing from bitcoind', async () => {
-  const initBitcoindBalance = await specterWallet.getBitcoindBalance()
-  const { chain_balance: initLndBalance } = await getChainBalance({ lnd })
-  
-  const sats = 5000
-  
-  await specterWallet.toLndWallet({ sats })
-  await bitcoindDefaultClient.generateToAddress(3, RANDOM_ADDRESS)
-  
-  const bitcoindBalance = await specterWallet.getBitcoindBalance()
-  
-  // const { chain_balance: lndBalance } = await getChainBalance({ lnd })
-  
-  // console.log({initBitcoindBalance, bitcoindBalance, lndBalance, initLndBalance})
-  // expect(bitcoindBalance).toBe(initBitcoindBalance - sats)
-  // expect(lndBalance).toBe(initLndBalance + sats)
-
-  
-  // const balance = await SpecterWallet.listWallets()
-  // expect(balance).toBe(0)
-})
+// TODO: Fix or remove. Expectations were commented out
```

Comment:
if there is no `expect()` the test is failing with the new setup?

Reply:
Tests pass with no expectations.